odl/tomo/analytic/filtered_back_projection.py

```diff
@@ -440,7 +441,12 @@ def fourier_filter(x):
     ramp_function = fourier.range.element(fourier_filter)
 
     # Create ramp filter via the convolution formula with fourier transforms
-    return fourier.inverse * ramp_function * fourier
+    if isinstance(ray_trafo.range.weighting, NoWeighting):
+        # Compensate for potentially unweighted range of the ray transform
+        weights = ray_trafo.range.cell_volume
```

Comment:
Would prefer if the ramp itself was weighted by this, that would improve performance